website/src/content/docs/docs/libraries/typespec-client-generator-core/reference/data-types.md

```diff
@@ -34,6 +34,20 @@ model Azure.ClientGenerator.Core.ClientOptions
 | service? | `Namespace` | The service that this client is generated for. If not specified, TCGC will look up the first parent namespace decorated with `@service` for the target.<br />The namespace should be decorated with `@service`. |
 | name?    | `string`    | The name of the client. If not specified, the default name will be `<Name of the target>Client`.                                                                                                                |
 
+### `ExternalType` {#Azure.ClientGenerator.Core.ExternalType}
+
+```typespec
+model Azure.ClientGenerator.Core.ExternalType
+```
+
+#### Properties
+
+| Name               | Type     | Description |
+| ------------------ | -------- | ----------- |
+| fullyQualifiedName | `string` |             |
```

Comment:
Do we need to fill out the descriptions? In particular, I think that for package/version these may or may not be respected by individual languages so it would be good to use language like "If supported" for these two properties.